website/docs/topics/retrieval_augmentation.md

```diff
@@ -53,6 +53,66 @@ ragproxyagent.initiate_chat(
 )  # search_string is used as an extra filter for the embeddings search, in this case, we only want to search documents that contain "spark".
 ```
 
+## Example Setup: RAG with Retrieval Augmented Agents with PGVector
```

Comment:
Would you like to add a link to the notebook in this section?

Reply:
I added the link on line 122, thank you for the suggestion.